### ISSUE TITLE
Add meta information to the code parser

### ISSUE THREAD

**Issue by kirovboris** [DevExpress/testcafe#2367]
### Are you requesting a feature or reporting a bug?

feature

### What is the current behavior?

We can't get meta infomration via `embeddingUtils` methods

### Specify your

* operating system:any
* testcafe version:.0.19.2
* node.js version:any

**Comment by gsachdev** [comment]
Hello @kirovboris any update regarding this change as this change is my prerequisite to start.

**Comment by kirovboris** [comment]
Hi @gsachdev,

We plan to implement this feature in the next version. We will be able to create a `dev` release with this changes once we merged it.

**Comment by kirovboris** [comment]
This functionality is released in `testcafe@0.22.0`. Now fixture and test descriptors have `meta` field:
```js
{
    name: 'Fixture',
    loc : <Location Object>;
    start: 0;
    end: 1;

    meta: {
        field: 'value'
    };

    tests: [];
}
```
